NewHorizons/Utility/NewHorizonExtensions.cs

```diff
@@ -188,6 +188,11 @@ public static GameObject InstantiateInactive(this GameObject original)
             return copy;
         }
 
+        public static GameObject Instantiate(this GameObject original)
```

Comment:
can u not just use InstantiateInactive? since it has a check for if the object is inactive and the only usages of this are when u know the object is already inactive

Reply:
Because I want it to be active

Reply:
oki. u do setactive tho right after so idgi but yeah good point